requirements/debug.txt

```diff
@@ -50,16 +50,13 @@ six==1.16.0
     # via asttokens
 stack-data==0.6.2
     # via ipython
-toml==0.10.2
+tomli==2.0.1
```

Comment:
This changes the package?

Reply:
It seems `ipdb` installs `toml` or `tomli` depending on other dependencies: https://github.com/gotcha/ipdb/blob/master/setup.py#L66

Reply:
Interesting.